### PR TITLE
add Organization model

### PR DESCRIPTION
**Pull request by Opeyem1a**

### CODE REVIEW

**Comment by Opeyem1a** [comment]
* **#31** <a href="https://app.graphite.dev/github/pr/Teamable-Analytics/teamable-backend/31?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @Opeyem1a and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: app/models/course.py [hunk 1]
```diff
@@ -1,7 +1,11 @@
 from django.db import models
 
 from app.models.base_models import BaseModel
+from app.models.organization import Organization
 
 
 class Course(BaseModel):
     name = models.CharField(max_length=500)
+    organization = models.ForeignKey(
+        Organization, null=True, blank=True, on_delete=models.SET_NULL
```
Comment (justino599) [inline app/models/course.py:10]: Why set null on delete?

**Review by justino599 — APPROVED** [review]

File: app/models/course.py (hunk 1 above)
Comment (keyvankhademi) [reply app/models/course.py:10]: I believe we don't necessarily need to have an organization for each course. The application is independent of any LMS and can be used without them.

**Review by keyvankhademi — APPROVED** [review]

File: app/models/course.py (hunk 1 above)
Comment (Opeyem1a) [reply app/models/course.py:10]: I concur with Keyvan

**Pull request merged**